DigitalExcellence/dex-frontend
# Implement multiple call to actions

## Description

In the original designs of the detail page, it was possible to add multiple call-to-action options. This pull request will implement these designs and functionality.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Add a project with 3 call-to-action.
2. Go to the overview page and check if the project has the call to action buttons with the URLs.
3. Edit your project and change a url and remove a call to action button.
4. Validate the changes came through.

## Link to issue

Closes: #454
